bcdaworker/queueing/manager/que_test.go

```diff
@@ -267,3 +263,57 @@ func TestStartAlrJob(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, models.JobStatusCompleted, alrJob.Status)
 }
+
+// Test alr cancel logic
```

Comment:
Now ALR and BFD share the same retry logic. This way we just need to cover one for both endpoints.